SimTracker/SiStripDigitizer/plugins/SiStripDigitizerAlgorithm.cc

```diff
@@ -66,7 +73,35 @@ SiStripDigitizerAlgorithm::SiStripDigitizerAlgorithm(const edm::ParameterSet& co
       theSiNoiseAdder(new SiGaussianTailNoiseAdder(theThreshold)),
       theSiDigitalConverter(new SiTrivialDigitalConverter(theElectronPerADC, PreMixing_)),
       theSiZeroSuppress(new SiStripFedZeroSuppression(theFedAlgo)),
-      APVProbabilityFile(conf.getParameter<edm::FileInPath>("APVProbabilityFile")) {
+      APVProbabilityFile(conf.getParameter<edm::FileInPath>("APVProbabilityFile")),
+      includeAPVSimulation_(conf.getParameter<bool>("includeAPVSimulation")),
+      apvBaselinesFile_tib1_(conf.getParameter<edm::FileInPath>("apvBaselinesFile_tib1")),
+      apvBaselinesFile_tib2_(conf.getParameter<edm::FileInPath>("apvBaselinesFile_tib2")),
```

Comment:
the use of arrays and loops would save a lot of typing here (and elsewhere in this pr)